rados/rados.go

```diff
@@ -125,11 +147,22 @@ func (c *RadosConn) MgrCommand(args [][]byte) (buffer []byte, info string, err e
 
 	ll = ll.WithField("conn", conn.GetInstanceID())
 
-	ll.Trace("start executing mgr command")
-
-	buffer, info, err = conn.MgrCommand(args)
-
-	ll.WithError(err).Trace("complete executing mgr command")
+	ch := make(chan bool, 1)
+	go func() {
+		ll.Trace("start executing mgr command")
+		buffer, info, err = conn.MgrCommand(args)
+		ch <- true
+	}()
```

Comment:
Ditto here.

Reply:
-resolving due to earlier code change- 🙂